repository/h.json

```diff
@@ -230,6 +230,17 @@
 				}
 			]
 		},
+		{
+			"name": "Hare Highlight",
```

Comment:
Language packages are simply named after the language they support. Any other words like "syntax" or "highlight" etc. are superfluous and misleading: a syntax/language package unlocks many more features in Sublime Text that just the highlighting and typically includes things like snippets etc. So you can just name your package "Hare".

Reply:
Updated name to "Hare" in package channel
Added LICENSE file to syntax/language package repository